<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicTTKFMRadioPlayWidget</class>
 <widget class="QWidget" name="MusicTTKFMRadioPlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络电台播放窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>525</width>
     <height>305</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>网络电台播放窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TTKClickedSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>525</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255, 255, 255, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MusicMarqueeWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>40</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicGifLabelWidget" name="gifLabel_1" native="true">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>130</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
  </widget>
  <widget class="MusicGifLabelWidget" name="gifLabel_2" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
  </widget>
  <widget class="MusicGifLabelWidget" name="gifLabel_3" native="true">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>130</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
  </widget>
  <widget class="MusicGifLabelWidget" name="gifLabel_4" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
  </widget>
  <widget class="MusicGifLabelWidget" name="gifLabel_5" native="true">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>130</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>265</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>265</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>265</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>30</y>
     <width>336</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
   <widget class="QTreeWidget" name="itemTree">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>15</y>
      <width>320</width>
      <height>251</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>playButton</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>label_2</zorder>
  <zorder>previousButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>positionLabel</zorder>
  <zorder>titleWidget</zorder>
  <zorder>gifLabel_1</zorder>
  <zorder>gifLabel_2</zorder>
  <zorder>gifLabel_3</zorder>
  <zorder>gifLabel_4</zorder>
  <zorder>gifLabel_5</zorder>
  <zorder>addButton</zorder>
  <zorder>deleteButton</zorder>
  <zorder>infoButton</zorder>
  <zorder>widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTKClickedSlider</class>
   <extends>QSlider</extends>
   <header>ttkclickedslider.h</header>
  </customwidget>
  <customwidget>
   <class>MusicMarqueeWidget</class>
   <extends>QWidget</extends>
   <header>musicmarqueewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicGifLabelWidget</class>
   <extends>QWidget</extends>
   <header>musicgiflabelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
